TrackletAlgorithm/TracktoFeature.h

```diff
@@ -0,0 +1,62 @@
+#ifndef TrackletAlgorithm_TracktoFeature_H__
+#define TrackletAlgorithm_TracktoFeature_H__
+
+#include "ap_fixed.h"
+#include "BDT.h"
+#include "parameters.h"
+#include "TrackQualityMemory.h"
+
+void FeatureTransform(const TTTrack& Track, input_arr_t transformed_features) {
+
+    //Proper track parameter representations tbc
+    //All track parameters are shown here through strictly only some are needed for this particular model
+
+    ap_int<15>      InvR        = Track.getInvR();
```

Comment:
In this and the following lines, could you use `auto` or to get rid of the magic number bit widths?